plugins/Installation/lang/en.json

```diff
@@ -19,145 +19,145 @@
+        "SystemCheckSummaryThereWereWarnings": "There are some issues with your system. Matomo will run, but you might experience minor problems.",
+        "SystemCheckTimeLimitHelp": "On a high-traffic website, executing the archival process may take longer than currently allowed. If necessary, change the 'max_execution_time' directive in your php.ini file.",
         "SystemCheckTracker": "Tracker status",
-        "SystemCheckTrackerHelp": "GET request to matomo.php failed. Try whitelisting this URL from HTTP Authentication and disable mod_security (you may have to ask your webhost). For more information about the error, check your web server error log file.",
-        "SystemCheckWarnDomHelp": "You should enable the \"dom\" extension (e.g., install the \"php-dom\" and\/or \"php-xml\" package).",
-        "SystemCheckWarning": "Matomo will work normally but some features may be missing",
-        "SystemCheckWarnJsonHelp": "You should enable \"json\" extension (e.g., install the \"php-json\" package).",
+        "SystemCheckTrackerHelp": "Could not perform GET request to matomo.php. Try whitelisting this URL from HTTP authentication and disable 'mod_security' (you may have to ask your webhost). More info about the error can be found in the error log-file on your web server.",
+        "SystemCheckWarnDomHelp": "You should turn on the \"dom\" extension (e.g., install the \"php-dom\" and\/or \"php-xml\" package).",
+        "SystemCheckWarning": "Matomo will work normally, but some features may be missing",
+        "SystemCheckWarnJsonHelp": "You should turn on the \"JSON\" extension (e.g., install the \"php-json\" package).",
         "SystemCheckWarnLibXmlHelp": "You should enable the \"libxml\" extension (e.g., \"install the php-libxml\" package) as it is required by other core PHP extensions.",
-        "SystemCheckWarnOpensslHelp": "You should enable the \"openssl\" extension to allow secure updates.",
+        "SystemCheckWarnOpensslHelp": "You should enable the \"OpenSSL\" extension to allow secure updates.",
         "SystemCheckWarnSimpleXMLHelp": "You should enable the \"SimpleXML\" extension (e.g., install the \"php-simplexml\" and\/or \"php-xml\" package).",
         "SystemCheckWinPdoAndMysqliHelp": "On a Windows server you can add the following lines to your php.ini: %s",
         "SystemCheckWriteDirs": "Directories with write access",
         "SystemCheckWriteDirsHelp": "To fix this error on your GNU\/Linux system, try typing in the following command(s)",
-        "SystemCheckZlibHelp": "You need to configure and rebuild PHP with \"zlib\" support enabled, --with-zlib.",
-        "SystemCheckCronArchiveProcess": "Setup Cron",
-        "FasterReportLoading": "faster report loading",
+        "SystemCheckZlibHelp": "Configure and rebuild PHP with the required \"zlib\" support, --with-zlib.",
+        "SystemCheckCronArchiveProcess": "Set up Cron",
+        "FasterReportLoading": "faster report-loading",
```

Comment:
Suggest reverting

Reply:
It isn't a faster report that is loading.